src/lib/plugin.js

```diff
@@ -104,15 +132,29 @@ module.exports = class PluginVirtual extends EventEmitter2 {
     this._rpc.addMethod('fulfill_condition', this._handleFulfillCondition)
     this._rpc.addMethod('reject_incoming_transfer', this._handleRejectIncomingTransfer)
     this._rpc.addMethod('expire_transfer', this._handleExpireTransfer)
-    this._rpc.addMethod('get_limit', this._handleGetLimit)
-    this._rpc.addMethod('get_balance', this._getBalance)
+
+    if (this._stateful) {
+      this._rpc.addMethod('get_limit', this._handleGetLimit)
+      this._rpc.addMethod('get_balance', this._getLowestBalance)
+      this._rpc.addMethod('get_info', () => Promise.resolve(this.getInfo))
+    }
+
     this.receive = this._rpc.receive.bind(this._rpc)
 
     // simple getters
     this.getInfo = () => JSON.parse(JSON.stringify(this._info))
     this.isConnected = () => this._connected
     this.getAccount = () => this._account
     this.isAuthorized = (authToken) => (authToken === this._authToken)
+
+    this._paychanBackend = PaymentChannelBackend || {}
```

Comment:
The existence of each method is checked before it's called; The empty object is put there in order to make sure that checking the truthiness of `_paychanBackend.something` won't cause an error.